src/SmartComponents/BaselinesPage/EditBaseline/EditBaselineKebab/EditBaselineKebab.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Dropdown, KebabToggle, DropdownItem } from '@patternfly/react-core';
+
+import DeleteFactModal from '../DeleteFactModal/DeleteFactModal';
+
+class EditBaselineKebab extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            kebabOpened: false,
+            modalOpened: false
+        };
+
+        this.toggleKebab = () => {
+            const { kebabOpened } = this.state;
+            this.setState({
+                kebabOpened: !kebabOpened
+            });
+        };
+
+        this.toggleModalOpened = () => {
+            const { modalOpened } = this.state;
+            this.setState({
+                modalOpened: !modalOpened,
+                kebabOpened: false
+            });
+        };
+    }
+
+    isFactSelected = () => {
+        const { editBaselineTableData } = this.props;
+        let isDisabled = true;
```

Comment:
I think this is mixing concerns. The function is all about figuring out if a fact is selected. This function is used by another chunk of the code to determine whether it disables something. But this function, shouldn't care about disabling or not, but rather is a fact selected. So I think changing this to `isSelected` would be cleaner.